pkg/engine/engine.go

```diff
@@ -377,14 +383,18 @@ func SupportsLineNumbers(sourceType sourcespb.SourceType) bool {
 }
 
 // FragmentLineOffset sets the line number for a provided source chunk with a given detector result.
-func FragmentLineOffset(chunk *sources.Chunk, result *detectors.Result) int64 {
+func FragmentLineOffset(chunk *sources.Chunk, result *detectors.Result) (int64, bool) {
 	lines := bytes.Split(chunk.Data, []byte("\n"))
 	for i, line := range lines {
 		if bytes.Contains(line, result.Raw) {
-			return int64(i)
+			// if the line contains the ignore tag, we should ignore the result
+			if bytes.Contains(line, []byte(ignoreTag)) {
```

Comment:
For the reviewer: this seemed like the most minimally invasive place to put the ignore logic. Alternative approaches could be adding a tag to the results metadata or abstracting a `location` struct similar to how gitleaks [handles locations](https://github.com/gitleaks/gitleaks/blob/master/detect/location.go) (admittedly a little less eloquently than trufflehog)